spec/system/candidate_interface/candidate_entering_gcse_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate entering GCSE details' do
+  include CandidateHelper
+
+  scenario 'Candidate submits their maths GCSE details' do
+    given_i_am_signed_in
+    and_i_visit_the_candidate_application_page
+    and_i_click_on_the_maths_gcse_link
+    then_i_see_the_add_gcse_maths_page
+
+    when_i_select_gcse_option
+    and_i_click_save_and_continue
+    then_i_see_add_grade_and_year_page
+
+    when_i_fill_in_grade_and_year
+    and_i_click_save_and_continue
+
+    then_i_see_the_review_for_maths_gcse
+    and_i_see_correct_grade_and_awarded_year
+  end
+
+  scenario 'Candidate does not provide a qualification level' do
```

Comment:
Elsewhere we've created separate files for separate scenarios. Might be a thing we want to do here.

Reply:
Not sure, this would require massive duplication, 
let's discuss for the following PR